test/nn/test_convolution.py

```diff
@@ -1749,6 +1750,29 @@ def test_conv2d_same_padding(self, device, dtype):
         actual = F.conv2d(x, y, padding="same", dilation=3)
         self.assertEqual(expect, actual, rtol=rtol, atol=atol)
 
+    @dtypes(torch.float)
+    # aten/src/ATen/native/mps/OperationUtils.mm: TORCH_INTERNAL_ASSERT([srcBuf length] > 0, "Placeholder tensor is empty!"); on MPS
+    @expectedFailureMPS
+    @skipXLA
+    def test_ConvTranspose_output_channels_0(self, device, dtype):
```

Comment:
```suggestion
    @skipCUDAIfRocm
    def test_ConvTranspose_output_channels_0(self, device, dtype):
```
I tried a few things to get this to work on ROCm but it's going to take longer to dig into it.  I updated the use_miopen check to avoid miopen when weight.size(1) == 0 but it falls through to hipblas and then fails there instead.